includes/classes/class-omise-card-image.php

```diff
@@ -12,7 +12,7 @@ class Omise_Card_Image {
 		 */
 		private static function get_image( $file, $alternate_text ) {
 			$url = WC_HTTPS::force_https_url( WC()->plugin_url() . '/assets/images/icons/credit-cards/' );
-			return "<img src='$url/$file' width='38px' alt='$alternate_text' />";
+			return "<img src='$url/$file' style='width: 38px;' alt='$alternate_text' />";
```

Comment:
@nimid ah, I meant, from this line,
```php
return "<img src='$url/$file' style='width: 38px;' alt='$alternate_text' />";
```
just add more class attr here
```php
return "<img src='$url/$file' class='OmiseCardBrandIcons' style='width: 38px;' alt='$alternate_text' />";
```

that customer can use `.OmiseCardBrandIcons` to override `style='width: 38px;'`,

---
alternate way,
have another css file to set 
```css
.OmiseCardBrandIcons {
  width: 38px;
}
```
instead of set style inline.